eclipse_plugin/.classpath

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
+	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
+	<classpathentry kind="src" path="src"/>
+	<classpathentry exported="true" kind="lib" path="lib/guava-19.0.jar"/>
```

Comment:
Guava 18+ is required due to usage of `MoreObjects`

    java.lang.NoClassDefFoundError: com/google/common/base/MoreObjects
	at com.google.googlejavaformat.java.JavaInput.characterRangeToTokenRange(JavaInput.java:563)
	at com.google.googlejavaformat.java.JavaInput.characterRangesToTokenRanges(JavaInput.java:641)

`MoreObjects.toStringHelper` and `MoreObjects.firstNonNull` are used.